.github/workflows/e2e.verify-token.schedule.yml

```diff
@@ -57,8 +60,24 @@ jobs:
           slsa-unverified-token: aGVsbG8K
           slsa-workflow-recipient: "delegator_generic_slsa3.yml"
           output-predicate: mismatch-token-predicate.json
+      - id: verify
```

Comment:
Give this a unique id?

Reply:
I removed this `uses`, not sure why it was present.